app/assets/javascripts/components/todo_list_container.js.jsx

```diff
@@ -7,19 +7,35 @@ Chk.TodoListContainer = (function () {
       todos: React.PropTypes.array.isRequired
     },
 
+    filterList: function(event){
+      if (event.target.checked){
```

Comment:
Here should be `!event.target.checked` (because the checkbox indicates if we want to show completed)